build/tools/DockerCompose.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Nuke.Common.IO;
+using Nuke.Common.Tooling;
+using Serilog;
+using Serilog.Core;
+
+namespace tools
```

Comment:
Nit: namespace should be `Pascal`

Reply:
I was trying to match the casing for the directories in this project which was Camel. I have adjusted as requested.